sdk/tests/deqp/functional/gles3/es3fLifetimeTests.js

```diff
@@ -435,10 +435,13 @@ es3fLifetimeTests.TfDeleteActiveTest.prototype.iterate = function() {
 
     gl.beginTransformFeedback(gl.TRIANGLES);
     var errCode = gl.NONE;
-    gl.deleteTransformFeedback(tf);
-    errCode = gl.getError();
-    assertMsgOptions(errCode == gl.INVALID_OPERATION,
-        'Deleting active transform feedback must produce INVALID_OPERATION', false, true);
+    try {
+        gl.deleteTransformFeedback(tf);
+    }
+    catch (err) {
+        assertMsgOptions(err.error == gl.INVALID_OPERATION,
```

Comment:
The only reason an exception is being thrown here is that in sdk/tests/deqp/functional/gles3/lifetime.html, create3DContextWithWrapperThatThrowsOnGLError is being used. I don't think it is a good idea to rely on this fact; that was mainly done for debugging. Please change that to wtu.create3DContext(...) instead and leave this line unchanged. Thanks.